### PR TITLE
String resource clean up

### PR DESCRIPTION
**Pull request by inverse**
- Consolidate string resources
- Make resources use consistent naming for ellipsis
- Make supported apps non-translatable, using a placeholder format to populate this correctly.

- readme, issues, etc changes from @HumbleBeeBumbleBee 

@HumbleBeeBumbleBee Does this cross over with your proposal on #204

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -1,28 +1,148 @@
...
-	
-	<string
-		name="will_submit_stuff">Enviará información a la página web
-	</string>
-	
-	<string
-		name="also_disable_np_title">También deshabilita el Escuchando ahora</string>
-	<string
-		name="also_disable_np_summary">[Hmpf]</string>
+    <string name="ok">Ok</string>
```
Comment (bryansills) [inline app/src/main/res/values-es/strings.xml]: Instead of having these strings, we should use Android's built-in strings for a few of these (ie. `android.R.string.ok`). There are a few more strings we could use here: http://developer.android.com/reference/android/R.string.html

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,28 +1,185 @@
...
+    <string name="auth_internal_error">Internal error</string>
+    <string name="auth_just_error">Error, retrying…</string>
+    <string name="logged_in_just">Logged in as:</string>
+    <string name="logged_in_as">Logged in as: %1</string>
+    <string name="log_in_to">Sign in to</string>
+    <string name="not_logged_in">Not logged in</string>
+    <string name="create_user_title">Sign up</string>
+    <string name="create_user_summary">Sign up for an account at %1</string>
+    <string name="confirm_clear_creds">Are you sure you want to clear your user
+        credentials for %1? This
```
Comment (bryansills) [inline app/src/main/res/values/strings.xml]: Why is this (and a few others) multi-line? It doesn't show up as multi-line in the actual app.

Comment (inverse) [inline app/src/main/res/values/strings.xml]: These were originally like this. I'll correct to be singular, I guess they were for readability but not necessary  :+1:

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,28 +1,147 @@
...
-	
-	<string
-		name="also_disable_np_title">Also disable now-playing</string>
-	<string
-		name="also_disable_np_summary">[Hmpf]</string>
+    <string name="close">Close</string>
+    <string name="remove">Remove</string>
+    <string name="reset">Reset</string>
+    <string name="clear_creds">Clear</string>
+    <string name="cancel">Cancel</string>
```
Comment (bryansills) [inline app/src/main/res/values/strings.xml]: Cancel is also in `android.R.strings`

Comment (inverse) [inline app/src/main/res/values/strings.xml]: Good catch :+1:

**Comment by HumbleBeeBumbleBee** [comment]
I was just wondering if you could put it like this 
```xml

    <!-- START of original strings-->
    <string name="close">Cerrar</string>
    <string name="remove">Eliminar</string>
    <string name="reset">Resetear</string>
    <string name="clear_creds">Limpiar</string>
    <string name="are_you_sure">¿Está seguro?</string>
    <string name="warning">Atención</string>
    <string name="will_submit_stuff">Enviará información a la página web</string>
    <string name="also_disable_np_title">También deshabilita el Escuchando ahora</string>
    <string name="also_disable_np_summary">[Hmpf]</string>
    <!-- END of original strings -->

    <!-- START of about strings -->
    <string name="by">por</string>
    <string name="issues">Incidencias:</string>
    <string name="website">Página web:</string>
    <string name="by_adam">por Adam Renberg</string>
    <string name="license">Liberado bajo licencia Apache 2.0</string>
    <string name="about_text">Un ligero cliente para hacer Scrobbling que envía información a Last.fm (y/o a Libre.fm) sobre qué música reproduces en tu móvil con Android. Last.fm te permite entonces ver estadísticas de lo escuchado y recomendarle nueva música que puede gustarle.</string>
    <string name="everything_disabled">Scrobbling/Escuchando ahora desactivado.</string>
    <string name="file_error">¡Error del archivo!</string>
    <string name="thats_new">Novedades</string>
    <string name="about">Acerca de</string>
    <string name="whats_new">Novedades</string>
    <string name="supported_netapps">Páginas webs soportadas: %1</string>
    <!-- END of about strings -->

    <!-- START of cache strings -->
```

**Comment by inverse** [comment]
Fixed merge conflicts :+1:

File: README.md [hunk 4]
```diff
@@ -234,12 +243,13 @@ Contributors
...
- * inverse [github.com/inverse](https://github.com/inverse), bug fixes, SoundCloud support
+ * inverse [github.com/inverse](https://github.com/inverse), bug fixes, maintenance
  * bryansills [github.com/bryansills](https://github.com/bryansills), Eclipse to Android Studio, new icon, Material Design
+ * herrherrmann [github.com/herrherrmann](https://github.com/herrherrmann), German translation
  
 Test device contributors
 ------------------
  * Dmitry Paskal, [github.com/paskal](https://github.com/paskal)
- * Iļja Gubins, [https://github.com/the-lay](https://github.com/the-lay)
+ * I?ja Gubins, [https://github.com/the-lay](https://github.com/the-lay)
```
Comment (bryansills) [inline README.md]: Undo this change.

**Comment by HumbleBeeBumbleBee** [comment]
Thanks @inverse !
